xmlStyle_test.go

```diff
@@ -69,9 +71,9 @@ func (x *XMLStyleSuite) TestMarshalXlsxStyleSheetWithABorder(c *C) {
 	border.Left.Style = "solid"
 	border.Top.Style = "none"
 	styles.Borders.Border[0] = border
-
 	expected := `<?xml version="1.0" encoding="UTF-8"?>
-<styleSheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><borders count="1"><border><left style="solid"></left><top style="none"></top></border></borders></styleSheet>`
+<styleSheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><borders count="1"><border><left style="solid"></left><right style=""></right><top style="none"></top><bottom style=""></bottom></border></borders></styleSheet>`
```

Comment:
There is no purpose to mixing -- it's a test that I missed I guess. They could all be "" or "none", and they may as well be "" for brevity and simplicity. Plus, it's what excel does.